clar2wasm/src/words/blockinfo.rs

```diff
@@ -18,6 +36,20 @@ impl ComplexWord for GetBlockInfo {
         expr: &SymbolicExpression,
         args: &[SymbolicExpression],
     ) -> Result<(), GeneratorError> {
+        if args.len() != 2 {
+            let (arg_name_offset, arg_name_len) =
+                generator.add_string_literal(&format!("expected: {} got: {}", 2, args.len()))?;
+            builder
+                .i32_const(arg_name_offset as i32)
+                .global_set(get_global(&generator.module, "runtime-error-arg-offset")?)
+                .i32_const(arg_name_len as i32)
+                .global_set(get_global(&generator.module, "runtime-error-arg-len")?)
+                .i32_const(ErrorMap::ArgumentCountMismatch as i32)
+                .call(generator.func_by_name("stdlib.runtime-error"))
+                // To avoid having to generate correct return values
+                .unreachable();
```

Comment:
You will generate correct return values since you don't return after the `unreachable`.

Reply:
Understood, I deleted it.